### PR TITLE
Docs: add a document for stress simulation

### PR DESCRIPTION
**Pull request by en-jin19**
Signed-off-by: en-jin19 <jinenwei@pingcap.com>

In this PR, I have added a document about stress simulation

This PR is translated from: https://github.com/chaos-mesh/website-zh/pull/52

### CODE REVIEW

File: docs/simulate-heavy-stress-in-physical-nodes.md [hunk 1]
```diff
@@ -1,6 +1,155 @@
 ---
```
Comment (en-jin19) [inline docs/simulate-heavy-stress-in-physical-nodes.md:1]: 哈喽王老师，麻烦也请帮忙看下这篇译文，谢谢！ @WangXiangUSTC

File: docs/simulate-heavy-stress-in-physical-nodes.md [hunk 2]
```diff
@@ -1,6 +1,155 @@
...
-title: 模拟压力场景
-sidebar_label: 模拟压力场景
+title: Simulate Stress Scenarios
+sidebar_label: Simulate Stress Scenarios
 ---
 
-## TODO
+This document describes how to use Chaosd to simulate stress scenarios. This feature generates CPU or memory sress on the host using [stress-ng](https://wiki.ubuntu.com/Kernel/Reference/stress-ng). You can create stress experiments either in command-line or service mode.
```
Comment (WangXiangUSTC) [inline docs/simulate-heavy-stress-in-physical-nodes.md:6]:
```suggestion
This document describes how to use Chaosd to simulate stress scenarios. This feature generates CPU or memory stress on the host using [stress-ng](https://wiki.ubuntu.com/Kernel/Reference/stress-ng). You can create stress experiments either in command-line or service mode.
```

File: docs/simulate-heavy-stress-in-physical-nodes.md [hunk 3]
```diff
@@ -1,6 +1,155 @@
...
+
+The result is as follows:
+
+```bash
+[2021/05/12 03:37:19.643 +00:00] [INFO] [stress.go:66] ["stressors normalize"] [arguments=" --vm 2 --vm-keep --vm-bytes 100000000"]
+[2021/05/12 03:37:19.654 +00:00] [INFO] [stress.go:82] ["Start stress-ng process successfully"] [command="/usr/bin/stress-ng --vm 2 --vm-keep --vm-bytes 100000000"] [Pid=26799]
+Attack stress mem successfully, uid: c2bff2f5-3aac-4ace-b7a6-322946ae6f13
+```
+
+When running the experiment, you need to save the uid information of the experiment.When a stress simulation is not needed, you can use `recover` to terminate the uid-related experiment:：
```
Comment (WangXiangUSTC) [inline docs/simulate-heavy-stress-in-physical-nodes.md:141]:
```suggestion
When running the experiment, you need to save the uid information of the experiment. When a stress simulation is not needed, you can use `recover` to terminate the uid-related experiment:：
```

**Review by WangXiangUSTC — COMMENTED** [review]
rest LGTM

**Review by WangXiangUSTC — APPROVED** [review]
LGTM

**Comment by en-jin19** [comment]
I have revised the translations according to the review, please kindly take a look, thanks! @WangXiangUSTC

**Review by cwen0 — APPROVED** [review]
LGTM

**Pull request merged**